websites/B/Blinest/presence.ts

```diff
@@ -0,0 +1,16 @@
+const presence = new Presence({
+	clientId: "828925279946801162",
+});
+
+presence.on("UpdateData", async () => {
+	presence.setActivity({
+		details: "Blinest",
+		state: "Playing on Blinest",
+		buttons: [
+			{
+				label: "Play",
+				url: "https://blinest.com/",
+			},
+		],
```

Comment:
Buttons that lead to home pages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets)
```suggestion
```